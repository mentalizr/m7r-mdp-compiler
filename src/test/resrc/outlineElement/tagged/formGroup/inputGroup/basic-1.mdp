@input-group[id=\"input_myId\"]
    Eine Ãœberschrift.

und hier folgt eine weitere Zeile ...