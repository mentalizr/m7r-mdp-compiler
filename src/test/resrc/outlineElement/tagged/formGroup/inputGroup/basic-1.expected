<div class="form-group">
    <label for="input_myId">Eine Ãœberschrift.</label>
    <input type="text" class="form-control ns_input" id="input_myId" placeholder="">
</div>